lib/ansible/modules/system/open_iscsi.py

```diff
@@ -187,6 +187,9 @@ def target_login(module, target):
                 module.fail_json(cmd=cmd, rc=rc, msg=err)
 
     cmd = '%s --mode node --targetname %s --login' % (iscsiadm_cmd, target)
+    if portal is not None and port is not None:
```

Comment:
Please fix one issue per commit. It will be easy to back-port and bisect changes while debugging.

Reply:
Thanks i agree completely. I didn't mean to include this, but I am new to git and I am trying to figure out how to separate the pull requests.

Reply:
No problem.

Reply:
This change has been removed.